keyboards/jisplit89/readme.md

```diff
@@ -14,4 +14,4 @@ Make example for this keyboard (after setting up your build environment):
 
 See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
 
-[Build guide](https://salicylic-acid3.hatenablog.com/entry/7skb-mx-build-guide)(See here because it is almost the same as 7sKB)
+[Build guide](https://salicylic-acid3.hatenablog.com/entry/jisplit89-build-guide)
```

Comment:
Is there a guide in English perhaps?

Reply:
> Is there a guide in English perhaps?

Some of the build guides he wrote seem to be all in Japanese.

https://salicylic-acid3.hatenablog.com/archive/category/%e3%83%93%e3%83%ab%e3%83%89%e3%82%ac%e3%82%a4%e3%83%89

Reply:
I have only written a guide in Japanese.

Reply:
I wouldn't mind taking a shot at translating this into English if I can get some down time.

Reply:
I'm not very good at English, so that's a very nice suggestion.
However, this keyboard has a Japanese layout, so few people may need the English build guide.